Microsoft.AspNetCore.Components.Server/Microsoft.AspNetCore.Components/ServerComponentMarker.cs

```diff
@@ -0,0 +1,80 @@
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.AspNetCore.Components
+{
+	internal struct ServerComponentMarker
+	{
+		public const string ServerMarkerType = "server";
+
+		public int? Sequence
+		{
+			[CompilerGenerated]
+			readonly get
+			{
+				throw null;
+			}
+			[CompilerGenerated]
+			set
+			{
+				throw null;
+			}
+		}
+
+		public string Type
+		{
+			[CompilerGenerated]
+			readonly get
+			{
+				throw null;
+			}
+			[CompilerGenerated]
+			set
+			{
+				throw null;
+			}
+		}
+
+		public string PrerenderId
```

Comment:
It looks like this should be nullable. We use that to indicate whether or not the component is prerendered ([ref](https://github.com/dotnet/aspnetcore/blob/master/src/Shared/Components/ServerComponentMarker.cs#L52))